File: TN-POS-SYS/TN-SYS-API/src/internal/model/m_db_guid_seq_itt.go
// 🇻🇳 Base Item Model với GUID và Sequence
// 🇺🇸 Base Item Model with GUID and Sequence
package model

// M_Db_Guid_Seq_Itm Base item với GUID ID và Sequence
type M_Db_Guid_Seq_Itm struct {
	M_Db_Guid_Itm
	QSeq int `json:"q_seq" gorm:"column:q_seq;default:1"`
}

File: TN-POS-SYS/TN-SYS-WEB/src/core/models/m_db_guid_seq_itt.ts
// 🇻🇳 Model cơ sở với GUID Identity và Sequence
// 🇺🇸 Base model with GUID Identity and Sequence
import type { M_Db_Guid_Itm } from './m_db_guid_itt';

export interface M_Db_Guid_Seq_Itm extends M_Db_Guid_Itm {
  // 🇻🇳 Số thứ tự hệ thống (Sequence)
  // 🇺🇸 System sequence number
  q_seq: number;
}

